x-pack/plugin/profiling/src/internalClusterTest/java/org/elasticsearch/xpack/profiling/ProfilingTestCase.java

```diff
@@ -81,7 +81,6 @@ protected final void createIndex(String name, String bodyFileName) throws Except
             .execute()
             .get();
         assertTrue("Creation of [" + name + "] is not acknowledged.", response.isAcknowledged());
-        assertTrue("Shards for [" + name + "] are not acknowledged.", response.isShardsAcknowledged());
```

Comment:
This lead to spurious test failures. As we wait later on until the cluster health is green there is no point in aggressively checking this here.